Automation/Redis_Cache/deploy.ps1

```diff
@@ -0,0 +1,27 @@
+
+# to run this, navigate to the Powershell  /Automation/Redis_Cache/deploy.ps1
+
+
+param(
+        [string] [Parameter(Mandatory=$true)] $SubscriptionId,
+        [string] [Parameter(Mandatory=$true)] $ResourceGroupName,
+        [string] [Parameter(Mandatory=$true)] $Location,
+        [string] [Parameter(Mandatory=$true)] $Redis_nsc_redis_dev_usw2_thursday_name
+
```

Comment:
Is it possible to remove the $SubscriptionId, $Location and $Redis_nsc_redis_dev_usw2_thursday_name  parameters since I don't see them used in this script?

Reply:
It would be better to just take them in as parameters and pass them in instead of the hard-coded values. Also I would rename the redis server name, like RedisServerName and just put "nsc_redis_dev_usw2_thursday" in the directions for testing.

Reply:
Thank you guys again for making these recommendations! I made changes based off your feedback and hopefully that does the trick.